pkg/antctl/antctl.go

```diff
@@ -129,12 +129,17 @@ var CommandList = &commandList{
 					params: []flagInfo{
 						{
 							name:  "name",
-							usage: "Get NetworkPolicy by name. If present, Namespace must be provided.",
+							usage: "Get NetworkPolicy by name.",
 							arg:   true,
 						},
+						{
+							name:      "source",
+							usage:     "Get Networkpolicies with specific source. Source means the name of its source network policy.",
```

Comment:
Can there be more than one internal NP for a given source?

Reply:
I think it is possible given the current implementation. cc @GraysonWu here as well.

Reply:
Yeah, internal NPs could have the same source(the name of its source np).

Reply:
The source name can be the same, but the source type and thus the source identity would be different. I think the first sentence should be rephrased: "Get NetworkPolicies for which the source has the provided name"

Reply:
Done, thanks.